selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -431,6 +431,18 @@ public String waitVisibilityAndGetValue(WebElement webElement) {
     return waitVisibility(webElement).getAttribute("value");
   }
 
+  /**
+   * Waits visibility of {@link WebElement} with provided {@code elementLocator} and gets text from
+   * its {@code value} value.
+   *
+   * @param elementLocator locator of element from which value should be got
+   * @param value name of element value
+   * @return element text by {@link WebElement#getAttribute(String)}
+   */
+  public String waitVisibilityAndGetValue(By elementLocator, String value) {
```

Comment:
It is `waitVisibilityAndGetAttribute` method, and `value` parameter here is actually `attributeName`.